features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Permission.js

```diff
@@ -150,6 +172,8 @@ class Permission extends Component {
         );
```

Comment:
add error handling. 

    promised_api.then(
              response => {
                this.msg.info(messages.success);
              }
          ).catch(
              error => {
                this.msg.error(messages.failure);
              }
          );

Reply:
it was already handled previously